src/CommunityToolkit.Aspire.Minio.Client/MinioClientBuilderExtensionMethods.cs

```diff
@@ -0,0 +1,100 @@
+﻿using CommunityToolkit.Aspire.Minio.Client;
+using Minio;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Provides extension methods for registering MiniO-related services in an <see cref="IHostApplicationBuilder"/>.
+/// </summary>
+public static class MinioClientBuilderExtensionMethods
+{
+    private const string DefaultConfigSectionName = "Aspire:Minio:Client";
+
+    /// <summary>
+    /// Adds Minio Client to ASPNet host
```

Comment:
```suggestion
    /// Registers <see cref="IMinioClient" /> as a singleton in the services provided by the <paramref name="builder"/>.
```

Or something to that effect.